targets/UnrealMarketplacePlugin/examplesource/ExampleProject/ExampleProject.uproject.ejs

```diff
@@ -9,5 +9,13 @@
             "Type": "Runtime",
             "LoadingPhase": "Default"
         }
-    ]
+    ],
+	"TargetPlatforms": [
```

Comment:
>"TargetPlatforms" [](start = 1, length = 17)

What about Xbox, PS4 and Switch? Is this section necessary for Android?

Reply:
I was getting errors without it, complaining that it couldn't find Android as a target (even tho the absence of this indicates "Any" target in the editor gui).  The section is auto-generated based on selections made in the editor gui,, I should add.  I don't recall the editor including those targets as options.  We should discuss tomorrow.

Reply:
I'm certain that Switch should not be in the targetplatforms that are submitted to the marketplace (because the plugin requires alternate code for that platform).
Otherwise, I'd go either way. I'd like them to be in the mkpl, but I don't care right away.